test/objectMode.js

```diff
@@ -0,0 +1,70 @@
+var tap = require('tap');
+var tape = require('../');
+var forEach = require('for-each');
+var through = require('through');
+
+tap.test('object results', function (assert) {
+    var printer = through({ objectMode: true });
```

Comment:
apparently, the `through` package has never supported an `objectMode` option, so i'm not sure what this is testing.

Reply:
objectMode does not exist as an option here, but the implementation is so trivial that it works with strings, objects & buffers by default. Where as node streams require opting in to allow objects with an objectMode flag.